apps/block_scout_web/lib/block_scout_web/templates/address/_responsive_hash.html.eex

```diff
@@ -5,8 +5,8 @@
     <%= if assigns[:truncate] do %>
       <%= BlockScoutWeb.AddressView.trimmed_hash(@address.hash |>  BlockScoutWeb.AddressView.loomnify) %>
     <% else %>
-      <span class="d-none d-md-none d-lg-inline"><%= @address |>  BlockScoutWeb.AddressView.loomnify %></span>
-      <span class="d-md-inline-block d-xl-none"><%= BlockScoutWeb.AddressView.trimmed_hash(@address.hash) %></span>
+      <span class="d-none d-lg-none d-xl-inline"><%= @address |>  BlockScoutWeb.AddressView.loomnify %></span>
+      <span class="d-lg-inline-block d-xl-none"><%= BlockScoutWeb.AddressView.trimmed_hash(@address.hash) %></span>
```

Comment:
I think this should be
```
<span class="d-lg-inline-block d-xl-none"><%= BlockScoutWeb.AddressView.trimmed_hash(@address.hash) |> BlockScoutWeb.AddressView.loomnify %></span>
```
And the `trimmed_hash` shouldn't be modified, it's also used to trim a tx hash in `overview.html.eex`, and tx hashes shouldn't have a `loom` prefix.